config/v3_0/types/verification.go

```diff
@@ -56,6 +56,8 @@ func (v Verification) Validate(c path.ContextPath) (r report.Report) {
 	switch function {
 	case "sha512":
 		hash = crypto.SHA512
+	case "sha256":
```

Comment:
I think we only want to do this for the upcoming 3.1.0, without retro-actively affecting 3.0.0.